config/layout.yml

```diff
@@ -15,8 +15,17 @@ Layout/LineLength:
   Description: Limit lines to 80 characters.
   Enabled: false
 
-# TODO: unclear why this is here!
-# https://github.com/alphagov/govuk-lint/commit/5e42ec5690e3d4cc8fff44d0b0268c8e581c423a
+# We have two styles of method call in our apps:
+#
+# SomeClass.first_method_call_on_same_line
+#          .other_method_call
+#
+# a_particularly_long_first_method_call
+#   .subsequent_method_call
+#
+# Any setting of this Cop would cause odd alignment
+# for one of these styles. Since there isn't a Cop to
+# set a preferred style, we should disable this one.
```

Comment:
Isn't this consistent with [the `EnforcedStyle: aligned` of `Layout/MultilineMethodCallIndentation`](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineMethodCallIndentation)?

Reply:
In the first case, yes. The second case is more nuanced than I thought. In our codebases, we have 

```
# before
a_particularly_long_first_method_call
  .subsequent_method_call

# after
a_particularly_long_first_method_call
.subsequent_method_call
```

This is a particularly ugly example/diff for Content Publisher.

```
       expect { described_class.call(params: params, user: user) }
         .to change { edition.reload.summary }.to("New summary")
-        .and change { edition.reload.change_note }.to("New note")
-        .and change { edition.reload.update_type }.to("minor")
+                                             .and change { edition.reload.change_note }.to("New note")
+                                                                                       .and change { edition.reload.update_type }.to("minor")
```

Does that help?